pkgs/by-name/bu/bulloak/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bulloak";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "alexfertel";
+    repo = "bulloak";
+    rev = "v${version}";
+    hash = "sha256-lUTMQMBqCezuUsfvuYSCBFsokoY3bPoJDGWL90EjVqY=";
+  };
+
+  cargoHash = "sha256-LH96e/dBbv4J7g7wzh3/vL+PzZn779zUMBgio6w3rJw=";
+
+  doCheck = false;
```

Comment:
Leaving a comment why tests disabled is always a good idea.

Reply:
Done